### PR TITLE
Prepare changelog for 0.97.0

### PR DESCRIPTION
**Pull request by atoulme**
This is a work in progress as we have yet to adopt contrib 0.97.0.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,14 +2,18 @@
...
 
+## v0.97.0
 
 ### 💡 Enhancements 💡
 
-- (Splunk) 'memory_ballast` has been removed. If GOMEMLIMIT env var is not set, then 90% of the total available memory limit is set by default. 
+- (Splunk) `memory_ballast` has been removed. If GOMEMLIMIT env var is not set, then 90% of the total available memory limit is set by default. ([#4404](https://github.com/signalfx/splunk-otel-collector/pull/4404))
+- (Splunk) Support Windows offline installations ([#4471](https://github.com/signalfx/splunk-otel-collector/pull/4471))
+- (Splunk) Add AWS container insights receiver ([#4125](https://github.com/signalfx/splunk-otel-collector/pull/4125))
+- (Splunk) Add AWS ECS container metrics receiver ([#4125](https://github.com/signalfx/splunk-otel-collector/pull/4125))
```
Comment (crobert-1) [inline CHANGELOG.md:12]: These should go under a new components header instead of enhancement.

**Review by samiura — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (atoulme) [reply CHANGELOG.md]: thanks moving

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,17 +2,31 @@
...
+
+- (Splunk) Add AWS container insights receiver ([#4125](https://github.com/signalfx/splunk-otel-collector/pull/4125))
+- (Splunk) Add AWS ECS container metrics receiver ([#4125](https://github.com/signalfx/splunk-otel-collector/pull/4125))
 
 ### 💡 Enhancements 💡
 
-- (Splunk) 'memory_ballast` has been removed. If GOMEMLIMIT env var is not set, then 90% of the total available memory limit is set by default. 
+- (Core) `configtls`: Validates TLS min_version and max_version ([#9475](https://github.com/open-telemetry/opentelemetry-collector/issues/9475))
+  Introduces `Validate()` method in TLSSetting.
+- (Splunk) `memory_ballast` has been removed. If GOMEMLIMIT env var is not set, then 90% of the total available memory limit is set by default. ([#4404](https://github.com/signalfx/splunk-otel-collector/pull/4404))
```
Comment (aunshc) [inline CHANGELOG.md:16]: Will start a separate discussion about this - our guidance in documentation and communication upfront (as a heads up) before this change goes out.

**Comment by atoulme** [comment]
https://github.com/signalfx/splunk-otel-collector/pull/4543 must be merged first.

File: CHANGELOG.md (hunk 2 above)
Comment (atoulme) [reply CHANGELOG.md:36]: https://github.com/signalfx/splunk-otel-collector/pull/4543 please review this PR for migration instructions.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,17 +2,73 @@
...
+### 🚀 New components 🚀
 
-- (Splunk) 'memory_ballast` has been removed. If GOMEMLIMIT env var is not set, then 90% of the total available memory limit is set by default. 
+- (Splunk) Add AWS container insights receiver ([#4125](https://github.com/signalfx/splunk-otel-collector/pull/4125))
+- (Splunk) Add AWS ECS container metrics receiver ([#4125](https://github.com/signalfx/splunk-otel-collector/pull/4125))
+- (Splunk) Add Apache metrics receiver ([#4505](https://github.com/signalfx/splunk-otel-collector/pull/4505))
+
+### 💡 Enhancements 💡
+
+- (Core) `configtls`: Validates TLS min_version and max_version ([#9475](https://github.com/open-telemetry/opentelemetry-collector/issues/9475))
```
Comment (crobert-1) [inline CHANGELOG.md:15]: Nit: We usually have Splunk changes first in the list

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,17 +2,73 @@
...
+    Add `compression` option to compress files using `compress/gzip` library before uploading to S3.
+- (Contrib) `awss3exporter`: Add support for encoding extension to awss3exporter ([#30554](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/30554))
+- (Contrib) `deltatocumulativeprocessor`: introduce configurable stream limit ([#31488](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31488))
+    Adds `max_streams` option that allows to set upper bound (default = unlimited)
+    to the number of tracked streams. Any additional streams exceeding the limit
+    are dropped.
+- (Contrib)`deltatocumulativeprocessor`: expire stale series ([#30705](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/30705), [#31016](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/31016))
+  Adds `max_stale` option that allows to set an interval (default = `5min`)
+  after which a series that no longer receives new samples is removed from
+  tracking.
```
Comment (crobert-1) [inline CHANGELOG.md:28]:
```suggestion
```
From what I can tell, we have the `cumulativetodeltaprocessor` in our distro, but not the `deltatocumulativeprocessor`.

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,17 +2,73 @@
...
 
 ### 🧰 Bug fixes 🧰
 
+- (Core) `exporterhelper`: Fix persistent queue size backup on reads.  ([#9740](https://github.com/open-telemetry/opentelemetry-collector/pull/9740))
+- (Core) `processor/batch`: Prevent starting unnecessary goroutines.  ([#9739](https://github.com/open-telemetry/opentelemetry-collector/issues/9739))
+- (Core) `otlphttpexporter`: prevent error on empty response body when content type is application/json  ([#9666](https://github.com/open-telemetry/opentelemetry-collector/issues/9666))
+- (Core) `otelcol`: Respect telemetry configuration when running as a Windows service  ([#5300](https://github.com/open-telemetry/opentelemetry-collector/issues/5300))
+- (Contrib) `carbonreceiver`: Do not report fatal error when closed normally ([#31913](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31913))
+- (Contrib) `deltatocumulativeprocessor`: timer-based expiry ([#31615](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/31615))
+    converts expiry to 1m timer, eliminating a race condition and failing test
```
Comment (crobert-1) [inline CHANGELOG.md:60]:
```suggestion
```
_Same comment as before_

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,17 +2,73 @@
...
+  The original metric `unit` does not apply to extracted `count` metrics the same way it does to `sum`, `min` or `max`.
+  Metrics extracted using `extract_count_metric()` now use the more appropriate default unit (`1`) instead.
+- (Contrib) `loadbalancingexporter`: Fix memory leaks on shutdown ([#31050](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31050))
+- (Contrib) `signalfxexporter`: Fix memory leak in shutdown ([#30864](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/30864), [#30438](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/30438))
+- (Contrib) `processor/k8sattributes`: Allows k8sattributes processor to work with k8s role/rolebindings when filter::namespace is set. ([#14742](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/14742))
+- (Contrib) `opencensusreceiver`: Refactor the opencensusreceiver to pass lifecycle tests and avoid leaking gRPC connections. ([#31643](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/31643))
```
Comment (crobert-1) [inline CHANGELOG.md:70]:
```suggestion
```
I don't think we have the opencensus receiver either.

**Pull request merged**